composer.json

```
@@ -2,7 +2,7 @@
     "name": "thg/thg-client",
     "description": "THG Open API client",
     "type": "library",
-    "version": "1.4.3",
+    "version": "2.1.3",
```

I wouldn't raise it to a new major version. There are no breaking changes, just an additional endpoint.
```suggestion
    "version": "1.5.0",
```